review.rake

```
@@ -15,6 +16,14 @@ task :remind, [:mode] do |_t, args|
   puts "-- DRY RUN --\n\n" if dry_run
 
   CONFIG.each do |app, app_config|
+    config_hours = app_config['notification_hours']
+    next unless config_hours
```

Should we `raise` instead of silently not doing anything?

---

Added a raise